ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -375,6 +375,7 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 				<Editor
 					value={props.content}
 					ref={editorRef}
+					parentSize={rootRef.current ? rootRef.current.getBoundingClientRect() : null}
```

Comment:
Shouldn't you just pass the width and height via the the style property? Then on useEffect, you just pass `[style.width, style.height]` as dependencies. That way you don't need to create a new property and we keep the one-way flow for props (i.e. parent sets props, and child doesn't need to know about parent props).

Reply:
Yes that's a good suggestion, I did this because it can be so succinct and in the update code it's very clear that the editor needs to refresh when the parent changes size. But I agree that it's probably the wrong method, I've updated now to pass the correct size to style and use that for updates.